SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -413,10 +467,33 @@ void USpatialSender::SendRPC(TSharedRef<FPendingRPCParams> Params)
 		// Target object was destroyed before the RPC could be (re)sent
 		return;
 	}
-
 	UObject* TargetObject = Params->TargetObject.Get();
+
+	if (AActor* TargetActor = Cast<AActor>(TargetObject))
+	{
+		USpatialNetConnection* SpatialConnection = NetDriver->GetSpatialOSNetConnection();
+		if (!NetDriver->GetActorChannelByEntityId(PackageMap->GetEntityIdFromObject(TargetActor)))
+		{
+			NetDriver->CreateActorChannel(TargetActor, SpatialConnection);
+		}
+
+		if (USpatialActorChannel* Channel = NetDriver->GetActorChannelByEntityId(PackageMap->GetEntityIdFromObject(TargetActor)))
```

Comment:
The GetActorByChannelByEntityId is duplicated here from the call above. Maybe try cleaning it up?

Reply:
Reorganized.